art/attacks/attack.py

```diff
@@ -145,20 +145,46 @@ def generate(self, x, y=None, **kwargs):
         raise NotImplementedError
 
 
-class PoisoningAttack(Attack):
+class PoisoningAttackBlackBox(Attack):
```

Comment:
Would it be useful to extend the docstring in line 150 similarly to the corresponding one for PoisoningAttackWhiteBox, e.g. mentioning no access to classifier?

Reply:
Sure thing. Fixed in commit.